refactor (#327)
KxSystems/docs: docs/architecture/rq.md

### .u.rep

> Initialise RDB by creating tables, which is then [populated with any existing tickerplant log](#recovery). Will set the HDB directory to use at end-of-day.

Reviewer: ```suggestion
Initialise RDB by creating tables, which is then [populated with any existing tickerplant log](#recovery). Sets the HDB directory to be used at end-of-day.
```
